code/drasil-example/glassbr/lib/Drasil/GlassBR/ModuleDefs.hs

```diff
@@ -163,7 +162,7 @@ extractColumnCT = funcDef "extractColumn" "Extracts a column from a 2D matrix"
   ]
 
 interpY :: Func
-interpY = funcDef (showUID U.interpY)
+interpY = funcDef (showSymb $ symbol U.interpY Implementation)
```

Comment:
These details should not be in GlassBR. The function that does
```haskell
foo :: HasSymbol x => x -> String
foo x = showSymb $ symbol x Implementation
```
should be in drasil-printers and then called from here.

Reply:
@JacquesCarette The requested change is added in 243618fd3324e77d864c839b82d6ba4b20b0af52